.github/actions/run-tests/action.yaml

```diff
@@ -24,3 +24,15 @@ runs:
     - name: Run sanity test
       shell: bash
       run: pushd ~/.ansible/collections/ansible_collections/dynatrace/oneagent && ansible-test sanity && popd
+    - name: Run integration tests
```

Comment:
Please rename to component tests or e2e tests. We do not test integration, the tests are isolated completely